src/container.tsx

```diff
@@ -66,12 +66,12 @@ export class Container extends React.Component<Properties> {
   }
   
   render() {
-    const { items, route: { app, znsRoute }, status, setSelectedItem, selectedItem } = this.props;
+    const { items, route: { app, znsRoute }, status, setSelectedItem, selectedItem, provider } = this.props;
     
     return (
       <>
         {isLeafNode(znsRoute, items) &&
-          <FeedLeafContainer item={selectedItem} />
+          <FeedLeafContainer item={selectedItem} provider={provider} />
```

Comment:
instead of passing the whole provider, let's just pull out the properties we need, and pass them explicitly. eg. if the feed leaf needs the current network, let's just pass that.